Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apt-get update && \
       gcc \
       vim-tiny \
       libffi-dev && \
-    pip install --upgrade pip
+    pip install --upgrade "pip<20.0"
```

Comment:
I think this is not needed anymore because 20.0.1 was released with a fix, right?